src/Tribe/Main.php

```diff
@@ -364,38 +369,80 @@ public function maybe_set_common_lib_info() {
 		}
 
 		/**
-		 * Plugins shouldn't include their functions before `plugins_loaded` because this will allow
-		 * better compatibility with the autoloader methods.
+		 * Resets the global common info back to ET's common path
 		 *
-		 * @return void
+		 * @since 4.9.3.2
 		 */
-		public function plugins_loaded() {
+		private function reset_common_lib_info_back_to_et() {
+			$et = Tribe__Tickets__Main::instance();
+			$et_common_version = file_get_contents( $et->plugin_path . 'common/src/Tribe/Main.php' );
+			$GLOBALS['tribe-common-info'] = [
+				'dir'     => "{$et->plugin_path}common/src/Tribe",
+				'version' => $et_common_version,
+			];
+		}
 
+		/**
+		 * Prevents bootstrapping and autoloading if the version of ET that is running is too old
+		 *
+		 * @since 4.9.3.2
+		 */
+		public function maybe_bail_if_old_et_is_present() {
 			// early check for an older version of Event Tickets to prevent fatal error
-			if (
-				class_exists( 'Tribe__Tickets__Main' ) &&
-				! version_compare( Tribe__Tickets__Main::VERSION, $this->min_et_version, '>=' )
-			) {
-				add_action( 'admin_notices', [ $this, 'compatibility_notice' ] );
-				add_action( 'network_admin_notices', [ $this, 'compatibility_notice' ] );
-				add_filter( 'tribe_ecp_to_run_or_not_to_run', [ $this, 'disable_pro' ] );
-				add_action( 'tribe_plugins_loaded', [ $this, 'remove_exts' ], 0 );
-				/*
-				* After common was loaded by another source (e.g. Event Tickets) let's append this plugin source files
-				* to the ones the Autoloader will search. Since we're appending them the ones registered by the plugin
-				* "owning" common will be searched first.
-				*/
-				add_action( 'tribe_common_loaded', [ $this, 'register_plugin_autoload_paths' ] );
-
-				// Disable older versions of Community Events to prevent fatal Error.
-				remove_action( 'plugins_loaded', 'Tribe_CE_Load', 2 );
+			if ( ! class_exists( 'Tribe__Tickets__Main' ) ) {
+				return;
+			}
+
+			if ( version_compare( Tribe__Tickets__Main::VERSION, $this->min_et_version, '>=' ) ) {
+				return;
+			}
+
+			$this->should_prevent_autoload_init = true;
+
+			add_action( 'admin_notices', [ $this, 'compatibility_notice' ] );
+			add_action( 'network_admin_notices', [ $this, 'compatibility_notice' ] );
+			add_filter( 'tribe_ecp_to_run_or_not_to_run', [ $this, 'disable_pro' ] );
+			add_action( 'tribe_plugins_loaded', [ $this, 'remove_exts' ], 0 );
```

Comment:
Not directly related to _this_ PR and possibly not a big concern: but going to note/ask anyway...won't this do more than just disable extensions? For instance, won't it stop Eventbrite from integrating with Aggregator (because that also happens at `tribe_plugins_loaded:10`) and so possibly other weirdness?

Reply:
A great question and I believe you are absolutely correct. I'll add this to my doc where I'm tracking what needs to be addressed when we revisit the Plugin Dependency work.